ffi/gn/build/BUILD.gn

```diff
@@ -0,0 +1,7 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+config("rust_defaults") {
+  rustflags = [ "-Cdebuginfo=2" ]
```

Comment:
Not sure what this is for, but it's in the rust GN example. Is this the right file to put it? The GN example doesn't have cross-compiling set up.